src/bots/PhindBot.js

```
@@ -33,155 +41,96 @@ export default class PhindBot extends Bot {
+    });
+    const onPhindRequest = (_, postData, text, resolve, reject) => {
+      try {
+        const source = new SSE("https://https.api.phind.com/agent/", {
+          start: false,
+          payload: postData,
+        });
+        source.addEventListener("message", (event) => {
+          if (event.data) {
+            /** @type {{ "created": number, "model": string, "choices": [ { "index": number, "delta": { "role": string, "content": string } } ] }} */
+            const response = JSON.parse(event.data);
+            if (response && response.choices && response.choices.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)